scm/driver/bitbucket/repo.go

```diff
@@ -86,6 +87,9 @@ func (s *repositoryService) FindPerms(ctx context.Context, repo string) (*scm.Pe
 // List returns the user repository list.
 func (s *repositoryService) List(ctx context.Context, opts scm.ListOptions) ([]*scm.Repository, *scm.Response, error) {
 	path := fmt.Sprintf("2.0/repositories?%s", encodeListRoleOptions(opts))
+	if opts.URL != "" {
+		path = strings.TrimPrefix(opts.URL, s.client.BaseURL.String())
```

Comment:
I think we can actually remove the `TrimPrefix` because when we try to combine the path with the base URL, if the path is a URL, it will return the URL. See https://play.golang.org/p/GXylQuPVrdx

So I think we can just simplify this to:

```
if opts.URL != "" {
		path = opts.URL
}
```

Reply:
Oh slick, I didn't release that was the behaviour

Reply:
me neither actually ... I just wondered how it would behave if we passed a full URL and was pleasantly surprised by the results :)

Reply:
Ok, updated in the last commit